examples/todomvc/src/components/MainSection.js

```diff
@@ -1,82 +1,40 @@
-import React, { Component } from 'react'
+import React from 'react'
 import PropTypes from 'prop-types'
-import TodoItem from './TodoItem'
 import Footer from './Footer'
-import { SHOW_ALL, SHOW_COMPLETED, SHOW_ACTIVE } from '../constants/TodoFilters'
-
-const TODO_FILTERS = {
-  [SHOW_ALL]: () => true,
-  [SHOW_ACTIVE]: todo => !todo.completed,
-  [SHOW_COMPLETED]: todo => todo.completed
-}
-
-export default class MainSection extends Component {
-  static propTypes = {
-    todos: PropTypes.array.isRequired,
-    actions: PropTypes.object.isRequired
-  }
-
-  state = { filter: SHOW_ALL }
-
-  handleClearCompleted = () => {
-    this.props.actions.clearCompleted()
-  }
-
-  handleShow = filter => {
-    this.setState({ filter })
-  }
-
-  renderToggleAll(completedCount) {
-    const { todos, actions } = this.props
-    if (todos.length > 0) {
-      return (
+import VisibleTodoList from '../containers/VisibleTodoList'
+
+const MainSection = (props) =>  {
+  const { todosCount, completedCount, actions } = props
```

Comment:
I would just destructure these inside the parameters of the function and have an implicit return.